### ISSUE TITLE
Fedora 24 installs PGHERO v1.4.1 by default, which does not work

### ISSUE THREAD

**Issue by raf64flo** [ankane/pghero#94]
Hello,

Using Centos7 repository (on a Fedora 24 distro), DNF installs an old version of pghero: 

```
$ uname -a
Linux urgi235 4.7.3-200.fc24.x86_64 #1 SMP Wed Sep 7 17:31:21 UTC 2016 x86_64 x86_64 x86_64 GNU/Linux

$ sudo dnf install pghero
Vérification de l'expiration des métadonnées effectuée il y a 1:02:40 sur Wed Oct  5 16:31:02 2016.
Dépendances résolues.
================================================================================================================================================================================================================================================================================
 Package                                                    Architecture                                               Version                                                                                 Dépôt                                                      Taille
================================================================================================================================================================================================================================================================================
Installation de:
 pghero                                                     x86_64                                                     1.4.1-1472180894.d4f0a70.centos7                                                        pghero                                                      33 M

Résumé de la transaction
================================================================================================================================================================================================================================================================================
Installation  1 Paquet

Taille totale des téléchargements : 33 M
Taille d'installation : 123 M
Est-ce correct [o/N] : o
Téléchargement des paquets :
pghero-1.4.1-1472180894.d4f0a70.centos7.x86_64.rpm                                                                                                                                                                                              6.5 MB/s |  33 MB     00:05    
--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
Total                                                                                                                                                                                                                                           6.5 MB/s |  33 MB     00:05     
Test de la transaction en cours
La vérification de la transaction a réussi.
Lancement de la transaction de test
Transaction de test réussie.
Exécution de la transaction
  Installation de               : pghero-1.4.1-1472180894.d4f0a70.centos7.x86_64                                                                                                                                                                                            1/1 
  Vérification                  : pghero-1.4.1-1472180894.d4f0a70.centos7.x86_64                                                                                                                                                                                            1/1 

Installé:
  pghero.x86_64 1.4.1-1472180894.d4f0a70.centos7                                                                                                                                                                                                                                

Terminé !

```

If i try to force the last version available, I get an error: 
```
$ sudo dnf install pghero-1.5.2-1475451201.f99a06d.centos7.x86_64 
Vérification de l'expiration des métadonnées effectuée il y a 1:06:25 sur Wed Oct  5 16:31:02 2016.
Erreur : nothing provides libpq5 needed by pghero-1.5.2-1475451201.f99a06d.centos7.x86_64
(essayez d'ajouter « --allowerasing » à la ligne de commande pour remplacer les paquets en conflit)
```
Even with the `--allow-erasing` option: 
```
$ sudo dnf install pghero-1.5.2-1475451201.f99a06d.centos7.x86_64 --allowerasing
Vérification de l'expiration des métadonnées effectuée il y a 1:07:39 sur Wed Oct  5 16:31:02 2016.
Erreur : nothing provides libpq5 needed by pghero-1.5.2-1475451201.f99a06d.centos7.x86_64
```

Anyway, if I follow the installation instructions, the service by default does nothing (it sleeps infinitely):
```
$ sudo pghero scale web=1
Scaling up...
Created symlink from /etc/systemd/system/pghero-web.service.wants/pghero-web-1.service to /etc/systemd/system/pghero-web-1.service.
--> done.


$ sudo service pghero-web-1  start
Redirecting to /bin/systemctl start  pghero-web-1.service

$ sudo service pghero status
Redirecting to /bin/systemctl status  pghero.service
● pghero.service
   Loaded: loaded (/etc/systemd/system/pghero.service; enabled; vendor preset: disabled)
   Active: active (running) since mer. 2016-10-05 17:44:04 CEST; 4s ago
 Main PID: 28247 (sleep)
    Tasks: 1 (limit: 512)
   CGroup: /system.slice/pghero.service
           └─28247 /bin/sleep infinity

oct. 05 17:44:04 urgi235 systemd[1]: Started pghero.service.

```

Then the curl command fails: 
```
$ curl -v http://localhost:3001/
*   Trying 127.0.0.1...
* connect to 127.0.0.1 port 3001 failed: Connection refused
* Failed to connect to localhost port 3001: Connection refused
* Closing connection 0
curl: (7) Failed to connect to localhost port 3001: Connection refused

```
Nothing appers in the logs: 
```
$ sudo pghero logs
==> /var/log/pghero/production.log <==
```

And what if I start the app directly: 
```
$ sudo /usr/bin/pghero run web
[29321] Puma starting in cluster mode...
[29321] * Version 3.4.0 (ruby 2.0.0-p648), codename: Owl Bowl Brawl
[29321] * Min threads: 1, max threads: 16
[29321] * Environment: production
[29321] * Process workers: 3
[29321] * Preloading application
[29321] ! Unable to load application: NameError: uninitialized constant ActionView::Helpers::ActiveModelHelper
/opt/pghero/vendor/bundle/ruby/2.0.0/gems/actionview-4.2.7.1/lib/action_view/helpers.rb:40:in `<module:Helpers>': uninitialized constant ActionView::Helpers::ActiveModelHelper (NameError)
        from /opt/pghero/vendor/bundle/ruby/2.0.0/gems/actionview-4.2.7.1/lib/action_view/helpers.rb:4:in `<module:ActionView>'
        from /opt/pghero/vendor/bundle/ruby/2.0.0/gems/actionview-4.2.7.1/lib/action_view/helpers.rb:3:in `<top (required)>'
        from /opt/pghero/vendor/bundle/ruby/2.0.0/gems/actionview-4.2.7.1/lib/action_view/base.rb:5:in `<top (required)>'
        from /opt/pghero/vendor/bundle/ruby/2.0.0/gems/actionview-4.2.7.1/lib/action_view/view_paths.rb:1:in `<top (required)>'
        from /opt/pghero/vendor/bundle/ruby/2.0.0/gems/actionpack-4.2.7.1/lib/abstract_controller/rendering.rb:4:in `<top (required)>'
        from /opt/pghero/vendor/bundle/ruby/2.0.0/gems/actionpack-4.2.7.1/lib/action_controller/base.rb:204:in `<class:Base>'
        from /opt/pghero/vendor/bundle/ruby/2.0.0/gems/actionpack-4.2.7.1/lib/action_controller/base.rb:164:in `<module:ActionController>'
        from /opt/pghero/vendor/bundle/ruby/2.0.0/gems/actionpack-4.2.7.1/lib/action_controller/base.rb:5:in `<top (required)>'
        from /opt/pghero/vendor/bundle/ruby/2.0.0/gems/pghero-1.4.1/app/controllers/pg_hero/home_controller.rb:2:in `<module:PgHero>'
        from /opt/pghero/vendor/bundle/ruby/2.0.0/gems/pghero-1.4.1/app/controllers/pg_hero/home_controller.rb:1:in `<top (required)>'
        from /opt/pghero/vendor/bundle/ruby/2.0.0/gems/railties-4.2.7.1/lib/rails/engine.rb:472:in `block (2 levels) in eager_load!'
        from /opt/pghero/vendor/bundle/ruby/2.0.0/gems/railties-4.2.7.1/lib/rails/engine.rb:471:in `each'
        from /opt/pghero/vendor/bundle/ruby/2.0.0/gems/railties-4.2.7.1/lib/rails/engine.rb:471:in `block in eager_load!'
        from /opt/pghero/vendor/bundle/ruby/2.0.0/gems/railties-4.2.7.1/lib/rails/engine.rb:469:in `each'
        from /opt/pghero/vendor/bundle/ruby/2.0.0/gems/railties-4.2.7.1/lib/rails/engine.rb:469:in `eager_load!'
        from /opt/pghero/vendor/bundle/ruby/2.0.0/gems/railties-4.2.7.1/lib/rails/engine.rb:346:in `eager_load!'
        from /opt/pghero/vendor/bundle/ruby/2.0.0/gems/railties-4.2.7.1/lib/rails/application/finisher.rb:56:in `each'
        from /opt/pghero/vendor/bundle/ruby/2.0.0/gems/railties-4.2.7.1/lib/rails/application/finisher.rb:56:in `block in <module:Finisher>'
        from /opt/pghero/vendor/bundle/ruby/2.0.0/gems/railties-4.2.7.1/lib/rails/initializable.rb:30:in `instance_exec'
        from /opt/pghero/vendor/bundle/ruby/2.0.0/gems/railties-4.2.7.1/lib/rails/initializable.rb:30:in `run'
        from /opt/pghero/vendor/bundle/ruby/2.0.0/gems/railties-4.2.7.1/lib/rails/initializable.rb:55:in `block in run_initializers'
        from /opt/pghero/vendor/ruby-2.0.0/lib/ruby/2.0.0/tsort.rb:150:in `block in tsort_each'
        from /opt/pghero/vendor/ruby-2.0.0/lib/ruby/2.0.0/tsort.rb:183:in `block (2 levels) in each_strongly_connected_component'
        from /opt/pghero/vendor/ruby-2.0.0/lib/ruby/2.0.0/tsort.rb:219:in `each_strongly_connected_component_from'
        from /opt/pghero/vendor/ruby-2.0.0/lib/ruby/2.0.0/tsort.rb:182:in `block in each_strongly_connected_component'
        from /opt/pghero/vendor/ruby-2.0.0/lib/ruby/2.0.0/tsort.rb:180:in `each'
        from /opt/pghero/vendor/ruby-2.0.0/lib/ruby/2.0.0/tsort.rb:180:in `each_strongly_connected_component'
        from /opt/pghero/vendor/ruby-2.0.0/lib/ruby/2.0.0/tsort.rb:148:in `tsort_each'
        from /opt/pghero/vendor/bundle/ruby/2.0.0/gems/railties-4.2.7.1/lib/rails/initializable.rb:54:in `run_initializers'
        from /opt/pghero/vendor/bundle/ruby/2.0.0/gems/railties-4.2.7.1/lib/rails/application.rb:352:in `initialize!'
        from /opt/pghero/config/environment.rb:5:in `<top (required)>'
        from config.ru:3:in `require'
        from config.ru:3:in `block in <main>'
        from /opt/pghero/vendor/bundle/ruby/2.0.0/gems/rack-1.6.4/lib/rack/builder.rb:55:in `instance_eval'
        from /opt/pghero/vendor/bundle/ruby/2.0.0/gems/rack-1.6.4/lib/rack/builder.rb:55:in `initialize'
        from config.ru:in `new'
        from config.ru:in `<main>'
        from /opt/pghero/vendor/bundle/ruby/2.0.0/gems/rack-1.6.4/lib/rack/builder.rb:49:in `eval'
        from /opt/pghero/vendor/bundle/ruby/2.0.0/gems/rack-1.6.4/lib/rack/builder.rb:49:in `new_from_string'
        from /opt/pghero/vendor/bundle/ruby/2.0.0/gems/rack-1.6.4/lib/rack/builder.rb:40:in `parse_file'
        from /opt/pghero/vendor/bundle/ruby/2.0.0/gems/puma-3.4.0/lib/puma/configuration.rb:313:in `load_rackup'
        from /opt/pghero/vendor/bundle/ruby/2.0.0/gems/puma-3.4.0/lib/puma/configuration.rb:242:in `app'
        from /opt/pghero/vendor/bundle/ruby/2.0.0/gems/puma-3.4.0/lib/puma/runner.rb:123:in `load_and_bind'
        from /opt/pghero/vendor/bundle/ruby/2.0.0/gems/puma-3.4.0/lib/puma/cluster.rb:370:in `run'
        from /opt/pghero/vendor/bundle/ruby/2.0.0/gems/puma-3.4.0/lib/puma/launcher.rb:172:in `run'
        from /opt/pghero/vendor/bundle/ruby/2.0.0/gems/puma-3.4.0/lib/puma/cli.rb:74:in `run'
        from /opt/pghero/vendor/bundle/ruby/2.0.0/gems/puma-3.4.0/bin/puma:10:in `<top (required)>'
        from /opt/pghero/vendor/bundle/ruby/2.0.0/bin/puma:23:in `load'
        from /opt/pghero/vendor/bundle/ruby/2.0.0/bin/puma:23:in `<main>'
```

Any hint?

**Comment by ankane** [comment]
From the error message, it seems like it doesn't like the `libpq5` dependency. However, I'm not super familiar with `dnf`, so not sure how to solve that. Hopefully someone else can weigh in.

**Comment by raf64flo** [comment]
In facts DNF is a replacement for YUM: https://fedoraproject.org/wiki/Features/DNF, so it should work in the same way, but since there is no package named `libpq5` the installation cannot proceed, even if I already have installed libpq shared object:
```
$ dnf provides libpq*
Vérification de l'expiration des métadonnées effectuée il y a 1:39:08 sur Wed Oct  5 16:33:27 2016.
postgresql-libs-9.5.4-1.fc24.x86_64 : The shared libraries required for any PostgreSQL clients
Dépôt               : @System

postgresql-server-9.5.4-1.fc24.x86_64 : The programs needed to create and run a PostgreSQL server
Dépôt               : @System

postgresql94-libs-9.4.9-1PGDG.f24.x86_64 : The shared libraries required for any PostgreSQL clients
Dépôt               : @System

postgresql-libs-9.5.4-1.fc24.i686 : The shared libraries required for any PostgreSQL clients
Dépôt               : updates

postgresql-libs-9.5.4-1.fc24.x86_64 : The shared libraries required for any PostgreSQL clients
Dépôt               : updates

postgresql-server-9.5.4-1.fc24.x86_64 : The programs needed to create and run a PostgreSQL server
Dépôt               : updates

libpqxx-1:4.0.1-6.fc24.i686 : C++ client API for PostgreSQL
Dépôt               : fedora

libpqxx-1:4.0.1-6.fc24.x86_64 : C++ client API for PostgreSQL
Dépôt               : fedora

libpqxx-devel-1:4.0.1-6.fc24.i686 : Development tools for libpqxx
Dépôt               : fedora

libpqxx-devel-1:4.0.1-6.fc24.x86_64 : Development tools for libpqxx
Dépôt               : fedora

libpqxx-doc-1:4.0.1-6.fc24.noarch : Developer documentation for libpqxx
Dépôt               : fedora

postgresql-libs-9.5.3-1.fc24.i686 : The shared libraries required for any PostgreSQL clients
Dépôt               : fedora

postgresql-libs-9.5.3-1.fc24.x86_64 : The shared libraries required for any PostgreSQL clients
Dépôt               : fedora

postgresql-server-9.5.3-1.fc24.x86_64 : The programs needed to create and run a PostgreSQL server
Dépôt               : fedora
```
```
$ dnf provides libpq5*
Vérification de l'expiration des métadonnées effectuée il y a 1:39:26 sur Wed Oct  5 16:33:27 2016.
Erreur : Aucune correspondance trouvée
```

Anyway, I get the same error than described in #32, even with the 1.4 version:
```
$ rpm -q pghero
pghero-1.4.1-1472180894.d4f0a70.centos7.x86_64
```

It seems what you thought to correct the problem last year didn't.

**Comment by ankane** [comment]
I'm able to reproduce with on CentOS 7 with the instructions in the doc. Will try to investigate when I have some time.

**Comment by admix** [comment]
Having similar issue with Centos 7:
```
--> Running transaction check
---> Package pghero.x86_64 0:1.5.2-1475686227.2bf4a7a.centos7 will be installed
--> Processing Dependency: libxslt1.1 for package: pghero-1.5.2-1475686227.2bf4a7a.centos7.x86_64
--> Processing Dependency: libpq5 for package: pghero-1.5.2-1475686227.2bf4a7a.centos7.x86_64
--> Finished Dependency Resolution
Error: Package: pghero-1.5.2-1475686227.2bf4a7a.centos7.x86_64 (pghero)
           Requires: libpq5
Error: Package: pghero-1.5.2-1475686227.2bf4a7a.centos7.x86_64 (pghero)
           Requires: libxslt1.1
 You could try using --skip-broken to work around the problem
 You could try running: rpm -Va --nofiles --nodigest
``` 

PS: should  there be another issue for Centos or can we modify the issue subject ? :-)

**Comment by ankane** [comment]
Just fixed (and confirmed in Vagrant). https://github.com/pghero/pghero/commit/39aecc8290d735423e438ad479e8496e84d84ca4

**Comment by raf64flo** [comment]
Great news! Thanks for the fix.

Currently, I have following versions available: 
```
$ dnf provides pghero
Vérification de l'expiration des métadonnées effectuée il y a 17:05:34 sur Wed Oct  5 16:33:27 2016.
pghero-1.0.0-1434160553.8283536.centos7.x86_64 : Containerized, standalone instance of PgHero
Dépôt               : pghero

pghero-1.0.0-1435304413.4285f6e.centos7.x86_64 : Containerized, standalone instance of PgHero
Dépôt               : pghero

pghero-1.1.0-1435304759.3de8a7c.centos7.x86_64 : Containerized, standalone instance of PgHero
Dépôt               : pghero

pghero-1.1.0-1435723795.a8ba2d0.centos7.x86_64 : Containerized, standalone instance of PgHero
Dépôt               : pghero

pghero-1.1.0-1435724100.41a868c.centos7.x86_64 : Containerized, standalone instance of PgHero
Dépôt               : pghero

pghero-1.2.0-1451550681.2218ff9.centos7.x86_64 : Containerized, standalone instance of PgHero
Dépôt               : pghero

pghero-1.2.0-1451551140.2218ff9.centos7.x86_64 : Containerized, standalone instance of PgHero
Dépôt               : pghero

pghero-1.2.0-1452065189.afb9e38.centos7.x86_64 : Containerized, standalone instance of PgHero
Dépôt               : pghero

pghero-1.2.0-1452162569.e5fe57c.centos7.x86_64 : Containerized, standalone instance of PgHero
Dépôt               : pghero

pghero-1.2.1-1453348497.e5fe57c.centos7.x86_64 : Containerized, standalone instance of PgHero
Dépôt               : pghero

pghero-1.2.2-1453348500.0ea06f8.centos7.x86_64 : Containerized, standalone instance of PgHero
Dépôt               : pghero

pghero-1.3.0-1467082950.37aca77.centos7.x86_64 : Containerized, standalone instance of PgHero
Dépôt               : pghero

pghero-1.3.2-1471329133.faaf826.centos7.x86_64 : Containerized, standalone instance of PgHero
Dépôt               : pghero

pghero-1.3.2-1471329140.faaf826.centos7.x86_64 : Containerized, standalone instance of PgHero
Dépôt               : pghero

pghero-1.3.2-1471924507.66479c5.centos7.x86_64 : Containerized, standalone instance of PgHero
Dépôt               : pghero

pghero-1.3.2-1471926319.7939450.centos7.x86_64 : Containerized, standalone instance of PgHero
Dépôt               : pghero

pghero-1.4.1-1472180744.f157a81.centos7.x86_64 : Containerized, standalone instance of PgHero
Dépôt               : pghero

pghero-1.4.1-1472180747.f157a81.centos7.x86_64 : Containerized, standalone instance of PgHero
Dépôt               : pghero

pghero-1.4.1-1472180894.d4f0a70.centos7.x86_64 : Containerized, standalone instance of PgHero
Dépôt               : pghero

pghero-1.4.1-1475298162.f065bde.centos7.x86_64 : Containerized, standalone instance of PgHero
Dépôt               : pghero

pghero-1.4.1-1475298728.ca9ebe7.centos7.x86_64 : Containerized, standalone instance of PgHero
Dépôt               : pghero

pghero-1.4.1-1475306589.258219e.centos7.x86_64 : Containerized, standalone instance of PgHero
Dépôt               : pghero

pghero-1.4.1-1475307172.9297427.centos7.x86_64 : Containerized, standalone instance of PgHero
Dépôt               : pghero

pghero-1.4.1-1475307328.e94217b.centos7.x86_64 : Containerized, standalone instance of PgHero
Dépôt               : pghero

pghero-1.5.2-1475385341.1b58b0e.centos7.x86_64 : Containerized, standalone instance of PgHero
Dépôt               : pghero

pghero-1.5.2-1475385345.1b58b0e.centos7.x86_64 : Containerized, standalone instance of PgHero
Dépôt               : pghero

pghero-1.5.2-1475390577.8be4e75.centos7.x86_64 : Containerized, standalone instance of PgHero
Dépôt               : pghero

pghero-1.5.2-1475390750.5bf39ef.centos7.x86_64 : Containerized, standalone instance of PgHero
Dépôt               : pghero

pghero-1.5.2-1475441364.8438c0a.centos7.x86_64 : Containerized, standalone instance of PgHero
Dépôt               : pghero

pghero-1.5.2-1475442758.2dccd48.centos7.x86_64 : Containerized, standalone instance of PgHero
Dépôt               : pghero

pghero-1.5.2-1475443287.2dccd48.centos7.x86_64 : A performance dashboard for Postgres
Dépôt               : pghero

pghero-1.5.2-1475444786.a27f5ee.centos7.x86_64 : A performance dashboard for Postgres
Dépôt               : pghero

pghero-1.5.2-1475445691.605ae83.centos7.x86_64 : A performance dashboard for Postgres
Dépôt               : pghero

pghero-1.5.2-1475447392.b777c62.centos7.x86_64 : A performance dashboard for Postgres
Dépôt               : pghero

pghero-1.5.2-1475448057.3287990.centos7.x86_64 : A performance dashboard for Postgres
Dépôt               : pghero

pghero-1.5.2-1475448395.10f65db.centos7.x86_64 : A performance dashboard for Postgres
Dépôt               : pghero

pghero-1.5.2-1475450473.febf01d.centos7.x86_64 : A performance dashboard for Postgres
Dépôt               : pghero

pghero-1.5.2-1475451201.f99a06d.centos7.x86_64 : A performance dashboard for Postgres
Dépôt               : pghero
```
I guess the `f99a06d` corresponds to the commit's SHA1, so according to that, RPM of the fix (`39aecc8`) is not yet available, right?

I don't know about the build system for RPM repository via [packager.io](https://rpm.packager.io/gh/pghero/pghero/centos7/master), could you inform me about when the fix will be available in the repo please?

**Comment by ankane** [comment]
I added Fedora 24 to the Vagrant config. https://github.com/pghero/pghero/commit/3cb941672eba18e6bd78b28f0d4a9d3dea2fde9a

The installation completes successfully (installs the latest version), but there's an error when running, which can be reproduced with:

```sh
sudo pghero run irb
require "nokogiri"
```

Fedora 24 isn't officially supported, so I won't be spending more time on this, but feel free to fork the repo and try to get it working, and I'll happily accept a PR.

**Comment by raf64flo** [comment]
It progresses. :)

```
$ sudo pghero run irb
irb(main):001:0> require "nokogiri"
LoadError: libgcrypt.so.11: cannot open shared object file: No such file or directory - /opt/pghero/vendor/bundle/ruby/2.3.0/gems/nokogiri-1.6.8.1/lib/nokogiri/nokogiri.so
	from /opt/pghero/vendor/ruby-2.3.1/lib/ruby/2.3.0/rubygems/core_ext/kernel_require.rb:55:in `require'
	from /opt/pghero/vendor/ruby-2.3.1/lib/ruby/2.3.0/rubygems/core_ext/kernel_require.rb:55:in `require'
	from /opt/pghero/vendor/bundle/ruby/2.3.0/gems/nokogiri-1.6.8.1/lib/nokogiri.rb:32:in `rescue in <top (required)>'
	from /opt/pghero/vendor/bundle/ruby/2.3.0/gems/nokogiri-1.6.8.1/lib/nokogiri.rb:28:in `<top (required)>'
	from /opt/pghero/vendor/ruby-2.3.1/lib/ruby/2.3.0/rubygems/core_ext/kernel_require.rb:127:in `require'
	from /opt/pghero/vendor/ruby-2.3.1/lib/ruby/2.3.0/rubygems/core_ext/kernel_require.rb:127:in `rescue in require'
	from /opt/pghero/vendor/ruby-2.3.1/lib/ruby/2.3.0/rubygems/core_ext/kernel_require.rb:40:in `require'
	from (irb):1
	from /opt/pghero/bin/irb:15:in `<main>'
```
The missing lib is provided by a package: 
```
$ dnf provides libgcrypt.so.11
Failed to synchronize cache for repo 'fedora-chromium-stable', disabling.
Last metadata expiration check: 0:09:06 ago on Thu Oct  6 22:06:28 2016.
compat-libgcrypt-1.5.5-5.fc24.i686 : A general-purpose cryptography library
Repo        : fedora-steam
```
Installation:
```
$ sudo dnf install -y compat-libgcrypt
Last metadata expiration check: 0:26:59 ago on Thu Oct  6 21:50:13 2016.
Dependencies resolved.
================================================================================================================================================================================================================================================================================
 Package                                                               Arch                                                        Version                                                              Repository                                                         Size
================================================================================================================================================================================================================================================================================
Installing:
 compat-libgcrypt                                                      x86_64                                                      1.5.5-5.fc24                                                         fedora-steam                                                      260 k

Transaction Summary
================================================================================================================================================================================================================================================================================
Install  1 Package

Total download size: 260 k
Installed size: 582 k
Downloading Packages:
compat-libgcrypt-1.5.5-5.fc24.x86_64.rpm                                                                                                                                                                                                        1.5 MB/s | 260 kB     00:00    
--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
Total                                                                                                                                                                                                                                           1.5 MB/s | 260 kB     00:00     
Running transaction check
Transaction check succeeded.
Running transaction test
Transaction test succeeded.
Running transaction
  Installing  : compat-libgcrypt-1.5.5-5.fc24.x86_64                                                                                                                                                                                                                        1/1 
  Verifying   : compat-libgcrypt-1.5.5-5.fc24.x86_64                                                                                                                                                                                                                        1/1 

Installed:
  compat-libgcrypt.x86_64 1.5.5-5.fc24                                                                                                                                                                                                                                          

Complete!
```
Then, the command you provide seems better: 
```
$ sudo pghero run irb
irb(main):001:0> require "nokogiri"
WARNING: Nokogiri was built against LibXML version 2.9.1, but has dynamically loaded 2.9.3
=> true
irb(main):002:0> 
```

I'll have to test further tomorrow because at home, pghero does not produces log file. o/

**Comment by raf64flo** [comment]
Ok, I finally got it!
On Fedora, package `compat-libgcrypt` is not available in main repositories, the tests I made yesterday at home used a Steam dedicated repository. At work I had to manually install a package from an unconfident repo: 
https://copr.fedorainfracloud.org/coprs/red/libgcrypt.so.11/
```
# don't use `dnf install` since the old package is not available for Fedora 24
sudo rpm -i https://copr-be.cloud.fedoraproject.org/results/red/libgcrypt.so.11/fedora-23-x86_64/00140779-compat-libgcrypt/compat-libgcrypt-1.5.3-4.fc23.x86_64.rpm
```
Then I was able to restart pghero and access web interface.

Well, I'll have to reinstall all of that on a CentOS server in order to avoid all that weird stuff.

Thanks for tips @ankane!
